tap_freshdesk/streams.py

```diff
@@ -285,6 +289,13 @@ class Conversations(ChildStream):
     path = 'tickets/{}/conversations'
     parent = 'tickets'
 
+    def add_fields_at_1st_level(self, record):
+        """
+        Overwrite updated__at value.
+        """
+        if record.get("last_edited_at"):
+            record["updated_at"] = max(record["updated_at"], record["last_edited_at"])
```

Comment:
Check if `updated_at` is available or not. Else it will throw TypeError if updated_at is None.

Reply:
`updated_at` is replicaiton key and that's why it is an `automatic key`. If it is `None` then tap should throw error. So, I think we should handle `None` value explicitly here.